cocos/core/renderer/ui/ui-batch-model.ts

```diff
@@ -73,8 +73,5 @@ class UISubModel extends SubModel {
     }
 
     public destroy () {
```

Comment:
Please delete the destroy function if no logic needed

Reply:
It is a public function, it may be used some where. So i kept this to avoid issues. And can add needed logic in future.